integration/spark2/src/main/scala/org/apache/spark/sql/CarbonEnv.scala

```diff
@@ -173,6 +174,10 @@ object CarbonEnv {
       .addListener(classOf[AlterTableDropPartitionPostStatusEvent],
         AlterTableDropPartitionPostStatusListener)
       .addListener(classOf[AlterTableDropPartitionMetaEvent], AlterTableDropPartitionMetaListener)
+      .addListener(classOf[AlterTableRenamePreEvent], LuceneRenameTablePreListener)
```

Comment:
Is this required? Ideally, lucene datamap is a separate module which should not have intrusive modification in other modules

Reply:
this listener class is added to block alter operation on lucene datamap, if we are blocking alter operation for all the datamaps, then this may not be required.